engine/table/src/main/java/io/deephaven/engine/table/impl/by/ByteChunkedVarOperator.java

```diff
@@ -99,8 +100,10 @@ private boolean addChunk(ByteChunk<? extends Values> values, long destination, i
                 final double variance = (newSum2 - (newSum * newSum / nonNullCount)) / (nonNullCount - 1);
                 resultColumn.set(destination, std ? Math.sqrt(variance) : variance);
             }
-        } else if (nonNullCounter.getCountUnsafe(destination) <= 1) {
+        } else if (nonNullCounter.getCountUnsafe(destination) == 1) {
             resultColumn.set(destination, Double.NaN);
+        } else if (nonNullCounter.getCountUnsafe(destination) == 0) {
+            resultColumn.set(destination, NULL_DOUBLE);
```

Comment:
It looks like the recent change to `CharChunkedVarOperator` should go in other places, like this.

Reply:
Arg, I forgot to replicate the operators.